Makefile

```diff
@@ -198,15 +217,18 @@ debug-shared: shared
 lib:
 	$(MKDIR) lib
 
-lib/libsass.a: $(COBJECTS) $(OBJECTS) | lib
-	$(AR) rcvs $@ $(COBJECTS) $(OBJECTS)
+lib/libsass.a: $(COBJECTS) $(OBJECTS) $(WASMOBJECTS) | lib
+	$(AR) rcvs $@ $(COBJECTS) $(OBJECTS) $(WASMOBJECTS)
```

Comment:
Not sure if this adds overhead in case we compile regular `libsass.a` (without wasm)?

Reply:
The ifdef at line 191 makes "wasm/libcxxabi_stubs.o" only available for the WASM. We need to place the libcxxabi_stubs.o in the wasm's lib/libsass.a since libc++abi is missing exception handling stuff (you can see it as a hack -- it is not possible to compile libsass without `-fno-exceptions`, which is right way to fix that for wasm atm)

Reply:
I see, I hope the expansion `$(WASMOBJECTS)` when `WASMOBJECTS` is undefined in OK then in the other case.